menu_generator.rb

```diff
@@ -0,0 +1,17 @@
+puts "Welcome to the Kreepy Kavern! \nour menu today is:\n "
+
+texture = ["Gellatinous", "Spongy", "Semi-solid", "Pulpy", "Crystallized",
+              "Frozen", "Condensed", "Liquid", "Loose", "Hot", "Fleshy",
+              "Plump", "Chunky", "Soupy", "Putrid"]
```

Comment:
I'd try to keep your indenting inline with the first line of the array, rather than with the other arrays. It makes it look  more consistent, but it's mostly a style preference.